### PR TITLE
163719 Update AWS stack creation script and AWS database backup for the DB Regression Test

### PR DESCRIPTION
**Pull request by MohitArora001**
Final AWS stack creation script and added RDS snapshot creation script

### CODE REVIEW

File: Scripts/aws_stack_provision.ps1 [hunk 1]
```diff
@@ -156,17 +154,17 @@ elseif ($EXISTING_INSTANCE_COUNT -eq 0){
       }
       Write-Host "CFN Stack $STACK_NAME is in CREATE_COMPLETE State"
       $PHYSICAL_INSTANCE_ID = ((Get-CFNStackResource -StackName $STACK_NAME -LogicalResourceId INSTANCE).PhysicalResourceId)
-      Write-Host "Trying to fetch VM Password"
+      ## It is recommended to wait 15 minutes before trying to fetch the VM password.
+      Write-Host "Waiting for 15 minutes before trying to fetch the VM password"
+      Start-Sleep -Seconds 900
```
Comment (robe070) [inline Scripts/aws_stack_provision.ps1]: No. Just loop until its succeeds.
And change `Start-Sleep` in function `fetch_vm_password` on line 33 to 20 seconds

File: Scripts/aws_stack_provision.ps1 [hunk 2]
```diff
@@ -196,6 +194,7 @@ function check_oracle_rds_status
    (
       [Parameter(Mandatory=$true)] [String]$ORACLE_DB_ID
    )
+   Write-Host "Checking Oracle RDS status"
```
Comment (robe070) [inline Scripts/aws_stack_provision.ps1]: What are you checking?

**Review by robe070 — CHANGES_REQUESTED** [review]

File: Scripts/aws_stack_provision.ps1 (hunk 2 above)
Comment (MohitArora001) [reply Scripts/aws_stack_provision.ps1]: If RDS is in Available state or not

File: Scripts/aws_stack_provision.ps1 (hunk 1 above)
Comment (MohitArora001) [reply Scripts/aws_stack_provision.ps1]: Updated the wait logic

**Comment by MohitArora001** [comment]
@robe070 Kindly review the final PR

**Review by robe070 — APPROVED** [review]

**Pull request merged**